modules/osdk-ansible-custom-resource-files.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-ansible.adoc
+
+[id='osdk-ansible-custom-resource-files-{context}']
+= Custom Resource files
+
+Operators use the Kubernetes' extension mechanism, Custom Resource Definitions
+(CRDs), so your Custom Resource (CR) looks and acts just like the built-in,
+native Kubernetes objects.
+
+The Custom Resource (CR) file format is a Kubernetes resource file. The object
+has mandatory and optional fields:
+
+.Custom Resource fields
+[cols="3,7",options="header"]
+|===
+|Field
+|Description
+
+|`apiVersion`
+|Version of the CR to be created.
+
+|`kind`
+|Kind of the CR to be created.
+
+|`metadata`
+|Kubernetes-specific metadata to be created.
+
+|`spec` (optional)
+|Key-value list of variables which are passed to Ansible. This field is empty by
+default.
+
+|`status`
+|Summarizes the current state of the object. For Ansible-based Operators, the
+link:https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#status-subresource[`status` subresource]
+is enabled for CRDs and managed by the `k8s_status` Ansible module by default,
+which includes `condition` information to the CR's `status`.
+
+|`annotations`
+|Kubernetes-specific annotations to be appended to the CR.
+|===
+
+The following list of Custom Resource (CR) annotations modify the behavior of
```

Comment:
Just wondering if you need to write out Custom Resource (CR) again, since you've already defined it above.

Reply:
Nope! Severe dupes in this section.

Overall, been trying to just write it out once per module, since I don't necessarily know if they'll be included in an assembly that has already defined it in the future.